src/x/debug/placement.go

```diff
@@ -51,8 +52,8 @@ func NewPlacementInfoSource(
 
 // Write fetches data about the placement and writes it in the given writer.
 // The data is formatted in json.
-func (p *placementInfoSource) Write(w io.Writer) error {
-	placement, _, err := p.getHandler.Get(p.serviceName, nil)
+func (p *placementInfoSource) Write(w io.Writer, httpReq *http.Request) error {
+	placement, _, err := p.getHandler.Get(p.serviceName, httpReq)
```

Comment:
Hey I was going to comment on this and say we really should just be flowing defaults all the way from the DB Node and/or the coordinator to all service related endpoints (as well as supporting the headers just in case you want to override it).

Anyway I went ahead and did this, now basically you don't need a special header, if you hit the DB node coordinator endpoint, or even just the default debug dump on the debug address HTTP server, it'll just return you whatever the current config is configured for:
https://github.com/m3db/m3/pull/1984

If you're good to go ahead with this, let's merge my pull request into your branch (by just using the merge button on that PR) and then maybe you could help fix up the tests and then get the resulting PR here merged into master?